PhpStormStubsMap.php

```diff
@@ -6986,7 +6988,44 @@ final class PhpStormStubsMap
   'DATE_RFC850' => 'date/date_d.php',
   'DATE_RSS' => 'date/date_d.php',
   'DATE_W3C' => 'date/date_d.php',
-  'DAY_1' => 'standard/standard_defines.php',
+  'DATE_W3C' => 'dio/dio_d.php',
```

Comment:
1) That's a duplicate array key (`DATE_W3C`)
2) Please adjust formatting. It is using 2 spaces by default, while you have this fragment indented with a tab.

Reply:
Removed